tests/entrypoints/openai/test_transcription_validation.py

```diff
@@ -60,6 +60,38 @@ async def test_basic_audio(mary_had_lamb, model_name):
         assert out_usage["seconds"] == 16, out_usage["seconds"]
 
 
+@pytest.mark.asyncio
+async def test_basic_audio_with_lora(mary_had_lamb):
+    """Ensure STT requests can pass LoRA requests through to generate."""
+    model_name = "ibm-granite/granite-speech-3.3-2b"
```

Comment:
do you want to replicate this test by checking translation in `test_translation_validation`?